projects/packages/codesniffer/Jetpack/Sniffs/PHPUnit/ClassNameSniff.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Sniff for PHPUnit's class name requirements.
+ *
+ * Eventually WPCS will add something like this, but we want it now.
+ *
+ * @see https://github.com/WordPress/WordPress-Coding-Standards/issues/2484
+ *
+ * @package automattic/jetpack-codesniffer
+ */
+
+namespace Automattic\Jetpack\Sniffs\PHPUnit;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Sniff for PHPUnit's class name requirements.
+ */
+class ClassNameSniff implements Sniff {
```

Comment:
I'm not familiar enough with Sniff conventions, but should this be more specific (e.g. `TestClassNameSniff`)?

Reply:
Sure, we can do that. I don't know to what extent there are conventions.